backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -23,6 +23,25 @@ This also includes situations where you have lost the majority of your control p
 +
 If applicable, you might also need to xref:../../backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-recovering-expired-certs[recover from expired control plane certificates].
 +
+[IMPORTANT]
+====
+You should only restore to an old etcd state when your etcd is not functioning and your kube-apiserver is inaccessible because etcd is down.
+If you can reach the kube-apiserver and get etcd content back, doing an etcd restore will almost certainly make your situation worse, not better.
```

Comment:
What do you mean by "get etcd content back"? How is that done?

Reply:
for laymen, perhaps

> If you can reach the kube-apiserver and get data back

If you can do something like `oc get crd` and get a response back with CRDs in it, then  it means your etcd is available.